data/data/manifests/openshift/cloud-creds-secret.yaml.template

```diff
@@ -25,6 +25,7 @@ data:
   azure_region: {{.CloudCreds.Azure.Base64encodeRegion}}
 {{- else if .CloudCreds.OpenStack}}
   clouds.yaml: {{.CloudCreds.OpenStack.Base64encodeCloudCreds}}
+  clouds.conf: {{.CloudCreds.OpenStack.Base64encodeCloudCredsINI}}
```

Comment:
Can you provide details on why we are including the `clouds.conf` in the secret that is used to talk to the API...

`clouds.conf` should be available from `openshift-config/cloud-provider-config` config map... ?